browser/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { NavigationMenu } from './view/components/Navbar/NavigationMenu';
+
+const navLinks = [
+  {
+    name: 'Logo',
+    path: '/',
+    Icon: () => (
+      <svg
+        xmlns='http://www.w3.org/2000/svg'
+        width='24'
+        height='24'
+        viewBox='0 0 24 24'
+        fill='none'
+        stroke='currentColor'
+        stroke-width='2'
+        stroke-linecap='round'
+        stroke-linejoin='round'
+        class='feather feather-circle nav-icon'
```

Comment:
React is going to freak out about some of these attributes.  class should be className.  stroke-linecap should be strokeLinecap (most likely... React should be throwing warnings).  Check out the browser console when you render this stuff.

Reply:
Gotcha, I'll make it a habit to check the console when I'm rendering something new. Will fix